src/components/forms/review-form.vue

```diff
@@ -20,33 +22,30 @@
         </div>
       </div>
       <div class="item">
-        <custom-picture
-          :alt="st.alt"
-          class="shadow"
-          :sources="st.sources"
-          :src="st.src"
-          :webp-sources="st.webpSources"
-        />
+        <slot name="second-token-image">
+          <custom-picture
+            :alt="image ? image.alt : ''"
+            class="shadow"
+            :sources="image ? image.sources : []"
+            :src="image ? image.src : ''"
+            :webp-sources="image ? image.webpSources : []"
+          />
+        </slot>
       </div>
     </div>
     <div class="items">
       <div class="item">
-        <h2>
-          {{ $t('treasury.increaseBoost.lblAmountWeDepositIn') }}
-          {{ token.symbol }}
-        </h2>
+        <h2>{{ `${inputAmountTitle} ${token.symbol}` }}</h2>
```

Comment:
This is bad for performance, it's better to use separate brackets instead
```suggestion
        <h2>{{ inputAmountTitle }} {{ token.symbol }}</h2>
```
Or if the text appears joined with `''` and it shouldn't do so, use `&nbsp;` symbol (non-breaking space)